### ISSUE TITLE
Sbas.export_geojson() problem 

### ISSUE THREAD

**Issue by georgeboldeanu** [mobigroup/gmtsar#152]
Hello, @AlexeyPechnikov ! I think that there is a problem with the sbas.export_geojson() function. The results from the function are a little bit odd, from a misalignment point of view. Exporting the same data as a raster with sbas.export_geotiff() and as a vector with sbas.export_geojson() provides 2 shifted outputs (See attachement).
![image](https://github.com/user-attachments/assets/41ccd881-9822-4ba0-a702-ba8ccf221e55)

 The raster is the original result exported **sbas.export_geotiff(displacement_ps_mm2[9:10], 'cop_fin_exps/displacement_ps_mm2_')** and the vector is exported with **sbas.export_geojson(displacement_ps_mm2[9:10],'cop_fin_exps/displacement_ps_mm2_vect_6iunie')**. 
In QGIS i didnt understand why the points from GeoJson were that concentrated, like 2points per pixel. I converted the raster to vector and it seems that there are 2 extra vectors (6 polygons per 4 cells). The red values are the values from the PyGMTSAR and the yellow values are extracted from SAGA GIS, same behaviour, 6 values on 4 cells. The red geometry is the converted raster to vector using basic QGIS functions.
What can cause such a behaviour?

**Comment by AlexeyPechnikov** [comment]
This is a QGIS issue with GeoTIFF files displaying: the shape of the files is distorted, so for example, a 5x5 pixel raster is shown as 4x6. You can resolve the problem by converting the GeoTIFF files to GeoNetCDF and opening them in QGIS without the issue:

```
gdal_translate disp.2015-03-10.tif disp.2015-03-10.nc
```

After that, you should see your GeoJSON points at the centers of the raster cells.

<img width="1920" alt="image" src="https://github.com/user-attachments/assets/ca9bc48e-07bf-4a88-b3c3-c36208e4837e">

**Comment by georgeboldeanu** [comment]
Yes, after doing gdal_translate the computed points where corectly put on the centroid of the pixels. The problem is that the pixels remain the same. Why is that, is there an asumption or any logic behind it?

**Comment by AlexeyPechnikov** [comment]
What problem?

**Comment by georgeboldeanu** [comment]
They have the shape from S1 original data, aprox 4x15. They are still in range x azimuth? I export the raster with export_geotiff function.  Also, from this there is a very high concentration of points..with some of them being duplicates, like 7.1 on the left side of the screenshot I provided. 
![image](https://github.com/user-attachments/assets/66d255c0-da23-47fa-8cd7-cb4cc76269d7)

**Comment by AlexeyPechnikov** [comment]
The results are, of course, exported in geographic coordinates, not radar coordinates. The grid spacing is chosen to be approximately equal to the average radar grid spacing. Note that radar pixels projected onto topography have very different shapes and areas. It is possible to produce the exact shapes of radar pixels in geographic coordinates, although these shapes can be complex. I have shared screenshots on LinkedIn and provided examples in the PyGMTSAR video lectures.

Rather than filtering out neighboring pixels with the same results, PyGMTSAR takes the opposite approach by merging them to produce precise shapes for homogeneously displaced areas.

<img width="1920" alt="Screenshot 2024-07-20 at 13 46 44" src="https://github.com/user-attachments/assets/45b8c0cf-b1c3-4e8f-9d05-b118e6ee5d5c">
